source/features/clean-conversation-headers.tsx

```diff
@@ -24,7 +24,8 @@ async function cleanPrHeader(byline: HTMLElement): Promise<void> {
 	// Extra author name is only shown on `isPRConversation`
 	// Hide if it's the same as the opener (always) or merger
 	const shouldHideAuthor = pageDetect.isPRConversation() && $('.author', byline)!.textContent === (await elementReady('.TimelineItem .author'))!.textContent;
-	if (shouldHideAuthor) {
+	const isSticky = byline.closest('.is-stuck') !== null;
```

Comment:
As you can see, the header will add a `is-stuck` class when it's sticky.

<img width="360" alt="image" src="https://github.com/user-attachments/assets/568bc265-18ea-49c6-ae38-16f5b4aa4409">

<img width="360" alt="image" src="https://github.com/user-attachments/assets/2983869d-8ee8-4c8c-b893-73b44ac63509">

Reply:
I’d prefer `gh-header-sticky` if it works because that’s what we generally use. The element exists (hidden) even if it’s not stuck